flocx_market/tests/unit/common/test_prepareservice.py

```diff
@@ -0,0 +1,58 @@
+import flocx_market.conf as conf
+import socket
+import flocx_market.common.service as flocx_market_service
+import pytest
+
+CONF = conf.CONF
+
+
+def test_service_defaults():
+    flocx_market_service.prepare_service()
+    assert(CONF.api.host_ip == "0.0.0.0")
+    assert(CONF.api.port == 8081)
+    assert(CONF.api.max_limit == 1000)
+    assert(CONF.api.public_endpoint is None)
+    assert(CONF.api.api_workers is None)
+    assert(CONF.api.enable_ssl_api is False)
+    assert(CONF.flask.SQLALCHEMY_TRACK_MODIFICATIONS is False)
+    assert(CONF.flask.PROPAGATE_EXCEPTIONS is False)
+    assert(CONF.host == socket.gethostname())
+
+    CONF.clear()
+
+
+def test_service_cli_valid():
+
+    args = ['dummy', '--api-port', '8085', '--api-host_ip', '0.0.0.1',
```

Comment:
I think that just testing a single option would be sufficient.

Reply:
I was testing to make sure erroneous arguments as well as incorrect argument types could not be passed into CONF, but if you feel this is unneeded I will remove it

Reply:
I agree that we need to test for incorrect arguments and invalid types, but that's not what this test is doing: it is checking that valid options work correctly. You address invalid options and types in subsequent tests. That said, I don't think you need to change this: it works fine, I think it's just longer than it needs to be. I'm happy with this one either way.